drools-eclipse/org.kie.eclipse/src/main/java/org/kie/eclipse/server/KieServiceDelegate.java

```diff
@@ -151,8 +152,7 @@ protected String httpDelete(String request) throws IOException {
 	protected String httpPost(String request, JsonObject body) throws IOException, RuntimeException {
 		String host = getKieRESTUrl();
 		URL url = new URL(host + "/" + request);
-		System.out.println("[POST] "+url.toString());
-		System.out.println("[POST] body: "+body);
+		System.out.println("[POST] "+url.toString()+" body: "+body);
```

Comment:
You're right, the sysouts should be removed - they were useful for
debugging but need to go now. I'll create another commit for this.

On Thu, Feb 18, 2016 at 1:12 PM, Kris <notifications@github.com> wrote:

> In
> drools-eclipse/org.kie.eclipse/src/main/java/org/kie/eclipse/server/KieServiceDelegate.java
> <https://github.com/droolsjbpm/droolsjbpm-tools/pull/42#discussion_r53375411>
> :
>
> > @@ -151,8 +152,7 @@ protected String httpDelete(String request) throws IOException {
> >  	protected String httpPost(String request, JsonObject body) throws IOException, RuntimeException {
> >  		String host = getKieRESTUrl();
> >  		URL url = new URL(host + "/" + request);
> > -		System.out.println("[POST] "+url.toString());
> > -		System.out.println("[POST] body: "+body);
> > +		System.out.println("[POST] "+url.toString()+" body: "+body);
>
> Are System.out statements still actually useful? Shouldn't they be removed
> or replaces by some more configurable logging?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/droolsjbpm/droolsjbpm-tools/pull/42/files#r53375411>.
>



-- 
________________________
Robert ("Bob") Brodt
Senior Software Engineer
JBoss by Red Hat